bootstrap.d/net-misc.y4.yml

```diff
@@ -109,6 +109,45 @@ packages:
       - args: ['cp', '@THIS_BUILD_DIR@/services', '@THIS_COLLECT_DIR@/etc/']
       - args: ['cp', '@THIS_BUILD_DIR@/protocols', '@THIS_COLLECT_DIR@/etc/']
 
+  - name: iperf3
+    labels: [aarch64, riscv64]
```

Comment:
Was this tested on those architectures?

Reply:
I don't have a tree that can run userspace for those, but given that this is a dependency-free C program I have little reason for doubt.